### PR TITLE
Fix 141, add option Export dynamically load resources

### PR DESCRIPTION
**Pull request by drelaptop**
Add a option, copy the dynamically load resources if `Export dynamically Load Resources` is selected, Like this

add option for plugin

What's the resources might be used in runtime? it's the resource located at `assets/resources` folder, I have confirmed this with @jareguo . 

Related Creator docs: English: [Acquire and load asset](http://docs.cocos.com/creator/manual/en/scripting/load-assets.html) 中文: [获取和加载资源
](http://docs.cocos.com/creator/manual/zh/scripting/load-assets.html#%E8%8E%B7%E5%8F%96%E5%92%8C%E5%8A%A0%E8%BD%BD%E8%B5%84%E6%BA%90)

### CODE REVIEW

**Comment by drelaptop** [comment]
Fix https://github.com/cocos2d/creator_to_cocos2dx/issues/141 , please review and merge @minggo

File: creator_project/packages/creator-luacpp-support/panels/setup-project-panel.js [hunk 1]
```diff
@@ -88,6 +88,12 @@ Editor.Panel.extend({
                     this.profileProject.save();
                 },
 
+                _onChangeexportDynamicallyLoadResource(event) {
```
Comment (minggo) [inline creator_project/packages/creator-luacpp-support/panels/setup-project-panel.js]: _onChangeexportDynamicallyLoadResource -> _onChangeExportDynamicallyLoadResource

File: creator_project/packages/creator-luacpp-support/panels/setup-project-panel.html [hunk 2]
```diff
@@ -21,10 +21,13 @@ <h2>Setup Target Project <span class="version">v{{version}}</span></h2>
         </div>
     </ui-prop>
 
-    <ui-prop name="Export Resource Only">
+    <ui-prop name="Export Resource">
         <ui-checkbox class="flex-1" v-value="profileProject.data.exportResourceOnly" v-on:confirm="_onChangeExportResourceOnly">
             Export Resource Only
         </ui-checkbox>
+        <ui-checkbox class="flex-1" v-value="profileProject.data.exportDynamicallyLoadResource" v-on:confirm="_onChangeexportDynamicallyLoadResource">
+            Export Dynamically Load Resource
```
Comment (minggo) [inline creator_project/packages/creator-luacpp-support/panels/setup-project-panel.html]: `Export Dynamically Load Resource` the meaning is not clear, you can ask Jason for some help.

Comment (drelaptop) [reply creator_project/packages/creator-luacpp-support/panels/setup-project-panel.html]: `Export Resource Dynamically Loaded` is a better choice, keep the same name style with `Export Resource Only`

**Comment by drelaptop** [comment]
please merge @minggo

**Pull request merged**